ibis/backends/flink/__init__.py

```diff
@@ -145,6 +145,34 @@ def list_tables(
         # but executing the SQL string directly yields a `TableResult` object
         return self._filter_with_like(tables, like)
 
+    def list_views(
+        self,
+        like: str | None = None,
+        temporary: bool = True,
+    ) -> list[str]:
+        """Return the list of view names.
+
+        Return the list of view names in the specified database and catalog.
+        or the default one if no database/catalog is specified.
+
+        Parameters
+        ----------
+        like : str, optional
+            A pattern in Python's regex format.
+        temporary : bool, optional
+            Whether to list temporary views or permanent views.
```

Comment:
This is exposing a lot of Flink stuff to users and maybe that's exactly what is needed, but does a user often need to distinguish between temporary and permanent views?  Or do they want to see the views in the current session / catalog?

Does it make sense to return the union of both?

Reply:
Good point. I've made this method internal/private now (also for an previous feedback), to avoid exposing the complexity to users.